ops/resources/virtual_network/_data.tf

```diff
@@ -0,0 +1,3 @@
+data "azurerm_resource_group" "rg" {
+  name = var.resource_group_name
```

Comment:
Is this just used to grab the location? Couldn't we just pass in the location variable instead?

Reply:
We could, yes! I just wanted to add a bit of additional robustness.